api/bases/keystone.openstack.org_keystoneapis.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.10.0
+    controller-gen.kubebuilder.io/version: v0.9.2
```

Comment:
It looks like your environment is using an older Golang/Operator-SDK than what we've recently upgraded to

Reply:
Looks like it uses old controller-gen in ./bin unless you clean it  and overwrites things. I'll see if I can fix it in another PR.

Reply:
https://github.com/openstack-k8s-operators/keystone-operator/pull/179